tsp-output/@azure-tools/typespec-csharp/src/Generated/Internal/ClientUriBuilder.cs

```diff
@@ -1,210 +1,82 @@
-// <auto-generated/>
+﻿// <auto-generated/>
 
-#nullable disable
+#nullable enable
```

Comment:
I don't have strong opinions about this implementation -- I just copied over the one from the customized OpenAI.

Reply:
Tracking bug fixes for this with this issue: https://github.com/Azure/autorest.csharp/issues/4480